net/libfab/libfab.c

```diff
@@ -489,7 +503,7 @@ static uint32_t libfab_handle_connect_request_events(struct m0_fab__tm *tm)
 			cd = (struct m0_fab__conn_data*)entry.data;
 			libfab_ep_ntop(cd->fcd_netaddr, &en);
 			libfab_fab_ep_find(tm, &en, cd->fcd_straddr, &ep);
-			rc = libfab_active_ep_create(ep, tm, entry.info);
+			rc = libfab_conn_accept(ep, tm, entry.info);
```

Comment:
result should be checked?

Reply:
We can only add an entry to the error log in this case. The error handling is done inside the libfab_conn_accept() API.
Added an entry to the error log.

Reply:
I don't understand, if error happens here, why does the function "m0_net_libfab_init()" still return 0?